portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,21 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFact(){
+    const facts =
+    [
+    'I am currently learning russian,Привет ', 
+    'I am interested in compilers and progamming languages',
+    'I have lived in three states: California, New York, and Virginia !'
+    ];
```

Comment:
style nit:

```
const facts = [
  "I am currently ...",
  "...",
  "...",
]

```
Since the array can't fit on a single line

https://google.github.io/styleguide/jsguide.html#formatting-block-indentation

Reply:
Done